modules/fluid_properties/doc/content/source/userobjects/LeadBismuthFluidProperties.md

```diff
@@ -0,0 +1,44 @@
+# LeadBismuthFluidProperties
+
+!syntax description /FluidProperties/LeadBismuthFluidProperties
+
+These properties are based on experiments reported in the Handbook on Lead-bismuth Eutectic Alloy and Lead
+Properties, Materials Compatibility, Thermal-hydraulics and Technologies [!citep](Fazio). 
+Most properties only depend on temperature, the fluid is considered incompressible.
+
+!table caption=Table of properties and references to the equations in [!citep](Fazio).
+| Properties                     | Equations| Equation # |
+| :----------------------------- | :------------- | :------------- |
+| Melting point $(T_{mo}, K)$    | 398 |  2.1  |
```

Comment:
mo is probably molten

Reply:
did (K) instead of [K] as moosedocs [k] is a link

Reply:
For future reference, square brackets can be escaped with backslashes. I actually wrote it this way, but Github decided to be "smart" and render it with Markdown so that you can't see it 😛